pkg/builds/dockerfile/build.go

```diff
@@ -101,6 +106,11 @@ type Options struct {
 // Build returns a TaskRun suitable for performing a Dockerfile build over the
 // provided source and publishing to the target tag.
 func Build(ctx context.Context, source name.Reference, target name.Tag, opt Options) *tknv1beta1.TaskRun {
+	context, args := RemoveArgument(opt.KanikoArgs, "context")
+	if context == "" {
+		// lets trim the leading / so that we can append easily to the /workspace/$(params.context) expression
+		context = strings.TrimPrefix(filepath.Dir(opt.Dockerfile), "/")
+	}
```

Comment:
With the new param, folks can just use `--context` without the `--kaniko-args` wrapper, so is this to bridge the gap for folks that may be passing it the old way?

Reply:
I read the linked issue, so I think I understand what you are after a bit more now.  looking again with fresh eyes...